docs/docs/exporting.md

```diff
@@ -53,18 +53,18 @@ loki-loki-ConfigMap
 loki-ingester-Service
 ```
 
-You can optionally use the template function `lower` for lower-casing fields, e.g. in the above example
+You can optionally use one of the following template functions: 
+ 
+- `lower` for lower-casing fields, e.g. in the above example
 
-```bash
-... --format='{{.metadata.labels.app}}-{{.metadata.name}}-{{.kind | lower}}'
-```
-
-would yield
-
-```
-loki-distributor-deployment
-```
+  ```bash
+  ... --format='{{.metadata.labels.app}}-{{.metadata.name}}-{{.kind | lower}}'
+  ```
+  would yield
 
-etc.
+  ```
+  loki-distributor-deployment
+  ```
+  etc.
 
-You can also use a different file extension by providing `--extension='yml'`, for example.
+You can also use a different file extension by providing `--extension='yml'`, for example.
```

Comment:
Unless I miss it, this is effectively not a change anymore, so please remove it :)

Reply:
Resolved